eclipse-scout-core/src/tile/TileGridLayoutConfig.ts

```diff
@@ -22,20 +22,42 @@ export interface TileGridLayoutConfigModel extends LogicalGridLayoutConfigModel
 /**
  * Configures layouting hints for tiles layouted by {@link TileGridLayout}.
  *
- * The configured hints only have an effect if theirs value is >=0.
- * Otherwise, the default values specified by CSS are applied (see {@link TileGridLayout._initDefaults}).
+ * @see LogicalGridLayoutConfig
  */
 export class TileGridLayoutConfig extends LogicalGridLayoutConfig implements TileGridLayoutConfigModel {
   declare model: TileGridLayoutConfigModel;
 
   maxWidth: number;
 
-  constructor(options?: InitModelOf<TileGridLayoutConfig>) {
-    super(options);
-    options = options || {};
-    if (options.maxWidth > -2) {
-      this.maxWidth = options.maxWidth;
+  protected static _DEFAULT_CONFIG: InitModelOf<TileGridLayoutConfig> = undefined;
+
+  constructor(options?: InitModelOf<TileGridLayoutConfig>, defaults?: InitModelOf<TileGridLayoutConfig>) {
+    super(options, defaults);
+    this._init(); // Needs to be initialized again otherwise maxWidth would be set to undefined
```

Comment:
Is that comment correct? $.extend() usually ignores properties that are undefined.